docs/developer-guide/meta/admin-api.md

```diff
@@ -0,0 +1,334 @@
+# Admin API
+
+The Admin API provides a simple way to view cluster information, including meta health detection, meta leader query, database metadata query, and datanode heartbeat detection. 
+
+The Admin API is an HTTP service that provides a set of RESTful APIs that can be called through HTTP requests. The Admin API is designed to be simple, easy to use and safe.
+
+Available API:
+
+- /health
+- /leader
+- /catalogs
+- /schemas
+- /tables
+- /heartbeat
+
+All these APIs are under the parent resource `/admin`.
+
+The following sections assume your meta instance is running on localhost port 3002.
+
+## /health HTTP endpoint  
+
+The `/health` endpoint accepts Get HTTP requests. Use this endpoint to check the health of your meta instance.
+
+### Definition
+
+```bash
+curl -X GET http://localhost:3002/admin/health
+```
+
+### Examples
+
+#### Request
+
+```bash
+curl -X GET http://localhost:3002/admin/health
+```
+
+#### Response
+
+```json
+OK
+```
+
+## /leader HTTP endpoint
+
+The `/leader` endpoint accepts Get HTTP requests. Use this endpoint to query the leader's addr of your meta instance.
+
+### Definition
+
+```bash
+curl -X GET http://localhost:3002/admin/leader
+```
+
+### Examples
+
+#### Request
+
+```bash
+curl -X GET http://localhost:3002/admin/leader
+```
+
+#### Response
+
+```json
+127.0.0.1:3002
+```
+
+## /catalogs HTTP endpoint
+
+The `/catalogs` endpoint accepts Get HTTP requests. Use this endpoint to query the all the catalog's name.
```

Comment:
```suggestion
The `/catalogs` endpoint accepts GET HTTP requests. Use this endpoint to query the all the catalog's name.
```

Reply:
@jun0315 Please take a look at this comment. And please help to fix the conflict when you have time.